src/routes/__test__/Router.test.tsx

```diff
@@ -0,0 +1,65 @@
+import { describe, expect, it, vi } from 'vitest'
+import { render, screen } from '@testing-library/react'
+import CssBaseline from '@mui/material/CssBaseline'
+import CustomThemeProvider from '@/theme/CustomThemeProvider'
+import Router from '../Router'
+import '@/i18n/config'
+
+// Mock all the route components
+vi.mock('../Home/Home', () => ({
+    default: () => <div data-testid="home-page">Home Page</div>
+}))
+
+vi.mock('../CodeOfConduct/CodeOfConduct', () => ({
+    default: () => <div data-testid="code-of-conduct-page">Code of Conduct Page</div>
+}))
+
+vi.mock('../Team/Team', () => ({
+    default: () => <div data-testid="team-page">Team Page</div>
+}))
+
+vi.mock('../JobBoard/JobBoard', () => ({
+    default: () => <div data-testid="job-board-page">Job Board Page</div>
+}))
+
+vi.mock('../ThemePreview/ThemePreview', () => ({
+    default: () => <div data-testid="theme-preview-page">Theme Preview Page</div>
+}))
+
+vi.mock('../NotFound/NotFound', () => ({
+    default: () => <div data-testid="not-found-page">Not Found Page</div>
+}))
+
+// Mock the BaseLayout component
```

Comment:
nit: prefer self-documenting code over comments, e.g. the code contains "mock" and "baseLayout" so the comment is less useful in this situation